release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -444,6 +444,13 @@ In {product-title} {product-version}, the installation program uses the Cluster
 
 In {product-title} {product-version}, the installation program uses the Cluster API instead of Terraform to provision clusters during installer-provisioned infrastructure installations on {azure-first} Stack Hub.
 
+[id="ocp-4-19-GCP-custom-dns_{context}"]
+==== Installing a cluster on {gcp-first} with user-provisioned DNS
+
+With this release, you can enable user-provisioned DNS instead of the default cluster-provisioned DNS solution. For example, your organization's security policies might not allow the use of public DNS services such as Google Cloud DNS. You can manage your DNS only for the IP addresses of the API and Ingress servers. If you use this feature, you must provide your own DNS solution that includes records for `api.<cluster_name>.<base_domain>.` and `*.apps.<cluster_name>.<base_domain>.`.
```

Comment:
```suggestion
With this release, you can enable user-provisioned domain name server (DNS) instead of the default cluster-provisioned DNS solution. For example, your organization's security policies might not allow the use of public DNS services such as Google Cloud DNS. You can manage your DNS only for the IP addresses of the API and Ingress servers. If you use this feature, you must provide your own DNS solution that includes records for `api.<cluster_name>.<base_domain>.` and `*.apps.<cluster_name>.<base_domain>.`.
```